tests/UuidBinaryOrderedTimeTypeTest.php

```diff
@@ -46,7 +47,7 @@ private function getType(): Type
 
     public function testGetName(): void
     {
-        $this->assertSame('uuid_binary_ordered_time', $this->getType()->getName());
+        $this->assertSame('uuid_binary_ordered_time', $this->getType()::lookupName($this->getType()));
```

Comment:
`lookupName()` was added in dbal 3.7.0: https://github.com/doctrine/dbal/pull/6130

It's not available at all with DBAL 2.